packages/web-components/src/components/footer/legal-nav.ts

```diff
@@ -119,18 +132,8 @@ class C4DLegalNav extends StableSelectorMixin(LitElement) {
         `;
   }
 
-  firstUpdated() {
-    const {
-      _adjunctLinksContainer: adjunctLinksContainer,
-      _adjunctLinksSlot: adjunctLinksSlot,
-    } = this;
-    const hideAdjunctLinksContainer =
-      adjunctLinksSlot?.assignedNodes().length === 0
-        ? adjunctLinksContainer?.classList.add(
-            `${prefix}--adjunct-links__container--hidden`
-          )
-        : '';
-    return hideAdjunctLinksContainer;
+  updated() {
+    this._handleAdjunctLinksVisibility();
```

Comment:
This updated method is actually probably removable now.

Reply:
Yes, it is removable. 
Addressed in my latest commit